src/RA_Integration.vcxproj

```diff
@@ -194,11 +229,18 @@
       <PreprocessorDefinitions>_DEBUG;%(PreprocessorDefinitions)</PreprocessorDefinitions>
     </ResourceCompile>
     <PostBuildEvent />
+    <PostBuildEvent>
+      <Command>IF "$(LocalDebuggerWorkingDirectory)"=="" GOTO done
+    IF NOT EXIST $(LocalDebuggerWorkingDirectory) GOTO done
+    echo Copying $(Configuration) DLL to $(LocalDebuggerWorkingDirectory)
+    copy /y $(TargetPath) $(LocalDebuggerWorkingDirectory)\$(TargetFileName)
+    copy /y $(TargetPath) $(LocalDebuggerWorkingDirectory)\$(TargetName).pdb
```

Comment:
This isn't what you want. You're creating a second copy *of the DLL* as the pdb file. You're not copying the PDB file.

Reply:
I don't know what magic your using but the pdb does not show up. Also the "_r" dll is just a backup it has nothing to do with the post-build.

The way you want:
![image](https://user-images.githubusercontent.com/18649925/40593466-c52a9880-61f6-11e8-8fc6-5c9cf9a55e88.png)

![image](https://user-images.githubusercontent.com/18649925/40593480-dace49fc-61f6-11e8-9918-67ed95a74192.png)

The way to get pdb:
![image](https://user-images.githubusercontent.com/18649925/40593524-091a16ec-61f7-11e8-8ea5-bdac7d4de77f.png)

![image](https://user-images.githubusercontent.com/18649925/40593563-4c74ee3a-61f7-11e8-9a97-48d663d7e0bd.png)

Reply:
![image](https://user-images.githubusercontent.com/32680403/40593630-fdadb9c2-61e6-11e8-97ca-9ad8f6b79ee8.png)

Reply:
You are totally right, I gotta fix it.

Reply:
This only affects Debug configurations and the ReleaseWithDebug configurations.